tools/darwin/Support/Codesign.command

```diff
@@ -9,7 +9,16 @@ export CODESIGN_ALLOCATE=`xcodebuild -find codesign_allocate`
 
 GEN_ENTITLEMENTS="$NATIVEPREFIX/bin/gen_entitlements.py"
 IOS11_ENTITLEMENTS="$XBMC_DEPENDS/share/ios11_entitlements.xml"
-LDID="$NATIVEPREFIX/bin/ldid"
+LDID32="$NATIVEPREFIX/bin/ldid32"
+LDID64="$NATIVEPREFIX/bin/ldid64"
+LDID=${LDID32}
+
+if [ "${CURRENT_ARCH}" == "arm64" ]; then
```

Comment:
`CURRENT_ARCH` isn't defined anywhere. This will break fake signing arm64 packages again, as the else branch is always executed.

Reply:
This script is an Xcode build phase script. Xcode sets this.

Reply:
Look into jenkins log, it's not set.

Reply:
Mhh i guess you mean in the case were Makefiles are used? Is this the case on Jenkins? If so whatever sets WRAPPER_NAME (which is also normally set by Xcode) needs to set CURRENT_ARCH aswell.

Reply:
Yes jenkins uses the cmake makefile generator.
If you mean the following it supports makefiles and xcode.
https://github.com/xbmc/xbmc/blob/a18701e7b2fe03780f90502d5d51e09cdc3ad718/cmake/scripts/osx/Macros.cmake#L2-L8

Reply:
Why again do we use a different build system on Jenkins then during development? When using Xcode for development something like this is very likely to stay unnoticed for a while. In my opinion this is suboptimal. We used xcodebuild before cmake introduction - but now it’s Makefiles I guess.

Reply:
That’s not what I meant - in the codesign script the variable WRAPPER_NAME is used. Something in cmake must set this for the Makefiles target - I will figure it out. Thx for catching that - would have been a pita to have this unnoticed.

Reply:
https://github.com/xbmc/xbmc/blob/9fcaef0ed9c4db8ca3775578a1e909ce21aee96d/cmake/scripts/ios/Install.cmake#L83